docker-compose-addons.yml

```diff
@@ -0,0 +1,198 @@
+version: '3'
+services:
+  cvmanager_api:
+    build:
+      context: ./services
+      dockerfile: Dockerfile.api
+    image: jpo_cvmanager_api:latest
+    restart: always
+    networks:
```

Comment:
I don't believe the 'networks' 'internal' tags are necessary - it looks like they are removed in the develop branch

Reply:
This was resolved by merging Develop in!